docs/installation.md

```diff
@@ -124,10 +124,10 @@ The default binary location for pipx-installed apps is `~/.local/bin`. This can
 variable `PIPX_BIN_DIR`. The default manual page location for pipx-installed apps is `~/.local/share/man`. This can be
 overridden with the environment variable `PIPX_MAN_DIR`.
 
-pipx's default virtual environment location is typically `~/.local/share/pipx` on Linux/Unix,
-`%USERPROFILE%\AppData\Local\pipx` on Windows and `~/Library/Application Support/pipx` on macOS, and for compatibility
-reasons, if `~/.local/pipx` exists, it will be used as the default location instead. This can be overridden with the
-`PIPX_HOME` environment variable.
+pipx's default virtual environment location is typically `~/.local/share/pipx` on Linux/Unix
+and `~/.local/pipx` on Windows and MacOS. For compatibility reasons, if `~/.local/pipx` on Linux, `%USERPROFILE%\AppData\Local\pipx`
+on Windows or `~/Library/Application Support/pipx` on MacOS exists, it will be used as the default location instead.
+This can be overridden with the `PIPX_HOME` environment variable.
```

Comment:
How about we suggest users setting PIPX_HOME to `~/.local/pipx` on Windows and macOS to avoid further issues?

Reply:
Updated it accordingly